dietpi/dietpi-software

```diff
@@ -7079,6 +7085,104 @@ _EOF_
 			/boot/dietpi/func/create_mysql_db wordpress wordpress "$GLOBAL_PW"
 		fi
 
+		if To_Install 141 # Hugo
+		then
+			hdir=/mnt/dietpi_userdata/hugo
+			G_AGI hugo
+			G_EXEC mkdir -p "$hdir"
+			Create_User -d "$hdir" hugo
+
+			G_EXEC cd "$hdir"
+			if ! [[ -d "${hdir}/mysite" ]]; then
+				G_EXEC hugo new site mysite -f toml
+			fi
+			G_EXEC cd "${hdir}/mysite"
+			if ! [[ -d themes/PaperMod ]]; then
+				Download_Install "https://github.com/adityatelange/hugo-PaperMod/archive/master.tar.gz" "${hdir}/mysite"
+				G_EXEC mv "${hdir}/mysite/hugo-PaperMod-master" "${hdir}/mysite/themes/PaperMod"
+			fi
+
+			## Create a default config with a default theme
+			cat << '_EOF_' > config.toml
+baseURL = 'http://my.website.com'
```

Comment:
As long as we do not know the hostname, it is probably better to use the IP address here? And is the port not required here for this to work correctly? But as the port is explicitly defined via CLI, it is probably smart enough to append it to the base URL automatically.
```suggestion
			cat << _EOF_ > config.toml
baseURL = 'http://$(G_GET_NET ip)'
```
or
```suggestion
			cat << _EOF_ > config.toml
baseURL = 'http://$(G_GET_NET ip):8191'
```

Reply:
It appends to the baseUrl automatically *unless* you specify `--appendPort=false` in the cli